### ISSUE TITLE
Importing Edward model module breaks Python command-line

### ISSUE THREAD

**Issue by esennesh** [blei-lab/edward#518]
When running Edward with tensorflow-1.0.1 (installed via python-pip) and Edward at commit cbdc3df2f5f1ba5be326d261ec4fab52d9a474a3, I observe the following:

    >>> import tensorflow as tf
    >>> from edward.models import Normal
    >>> W_0 = Normal(mu=tf.zeros([1, 2]), sigma=tf.ones([1, 2]))
    >>> W_0
    >>> W_0 is None
    >>>

I've also checked this with a simpler example:

    >>> import tensorflow as tf
    >>> from edward.models import Normal
    >>> e
    >>>

In short, once we import the Edward model module, everything from printing out result values to detecting undefined variables goes silent.  This is in a Python 2.7.12 prompt, but also appears to occur with Python 3.5.2.

**Comment by AidanRocke** [comment]
I'm encountering a similar problem. In particular I'm getting the following error:

`TypeError: First parent class declared for Empirical must be Distribution, but saw 'RandomVariable'`

The problem appears to be in line 13 here: https://github.com/blei-lab/edward/blob/9f53d0cde82c6e7b70286a023eaa881b6b883bbf/edward/models/random_variables.py

**Comment by dustinvtran** [comment]
@esennesh | Thanks for the report. I'm having trouble reproducing the bug unfortunately; have you tried the newest version release (Edward v1.2.3)? It's pip installable.

@AidanRocke | I think that should be fixed already. You need to use at least TensorFlow v1.0.0 and Edward v1.2.1; also see #315.

**Comment by AidanRocke** [comment]
@dustinvtran Thanks. It looks like this particular problem is solved.

**Comment by esennesh** [comment]
@dustinvtran, yes, this happens when I use the version available on pip (both pip2 and pip3) as well.

**Comment by dustinvtran** [comment]
I think I tracked down the error.

It comes from the following: https://github.com/blei-lab/edward/blob/master/edward/util/graphs.py#L11-L18. The immediate fix is to install Keras; if the user doesn't have Keras, the control flow doesn't rewrite the sys.stderr. I'll fix this immediately.

**Comment by dustinvtran** [comment]
@esennesh | Did that fix it? Let me know otherwise.

**Comment by esennesh** [comment]
@dustinvtran fixed!  Thanks a ton!
